README.md

```diff
@@ -22,54 +22,63 @@ J. Blow.
 ### Setup & Usage (For Repo Devs Only)
 
 Required:
-- [NodeJS](https://nodejs.org/)
-- Ocaml 4.06.1
-- OS: Mac
+
+- OCaml 4.06.1
+- Dune
+- Reanalyze
+- OS: macOS, Linux or Windows (tests will currently run on macOS only)
 
 ```sh
-opam switch create 4.06.1 && eval $(opam env)
+opam switch create 4.06.1 # Note: on macOS ARM, do "arch -x86_64 zsh" first
+eval $(opam env)
+opam install dune reanalyze
 git clone https://github.com/rescript-lang/syntax.git
 cd syntax
-npm install
-make # or: make -j9 for faster build
+make # or "dune build"
 ```
 
-This will produce the final binary `lib/rescript.exe` used for testing.
+This will produce the three binaries `rescript`, `tests` and `bench` (with `.exe` extension on Windows).
 
-We only build production binary, even in dev mode. No need for a separate dev binary when the build is fast enough. Plus, this encourages proper benchmarking of the (production) binary each diff.
+We only build production binaries, even in dev mode. No need for a separate dev binary when the build is fast enough. Plus, this encourages proper benchmarking of the (production) binary each diff.
 
 After you make a change:
+
 ```sh
 make
 ```
 
 Run the core tests:
+
 ```sh
 make test
 ```
 
 Run the extended tests:
+
 ```sh
 make roundtrip-test
 ```
 
 Those will tell you whether you've got a test output difference. If it's intentional, check them in.
 
 Debug a file:
+
 ```sh
 # write code in test.res
-./lib/rescript.exe test.res # test printer
-./lib/rescript.exe -print ast test.res # print ast
-./lib/rescript.exe -print ml test.res # show ocaml code
-./lib/rescript.exe -print res -width 80 test.res # test printer and change default print width
+dune exec -- rescript test.res # test printer
```

Comment:
Should there been Makefile shortcuts for these?

Reply:
I don't think we can pass parameters to Makefile targets like we do for npm scripts?

Reply:
Oh that's right, ignore this.